### ISSUE TITLE
Speech Note crashes on start

### ISSUE THREAD

**Issue by mutageneral** [mkiol/dsnote#41]
I'm on OpenSUSE Tumbleweed and I'm using the Flatpak version of Speech Note.

```
$ flatpak run net.mkiol.SpeechNote 
Qt: Session management error: Could not open network socket
ALSA lib ../../oss/pcm_oss.c:397:(_snd_pcm_oss_open) Cannot open device /dev/dsp
ALSA lib ../../../src/pcm/pcm_direct.c:2045:(snd1_pcm_direct_parse_open_conf) The field ipc_gid must be a valid group (create group audio)
ALSA lib ../../../src/pcm/pcm_direct.c:2045:(snd1_pcm_direct_parse_open_conf) The field ipc_gid must be a valid group (create group audio)
free(): invalid size
```

**Comment by mkiol** [comment]
Thank you for the report.

Could you please try to run with `--verbose` option?

```
$ flatpak run net.mkiol.SpeechNote --verbose
```

**Comment by caslubas** [comment]
I'm also using opensuse tumbleweed and I get the following log when I run the command with the verbose option:

`flatpak run net.mkiol.SpeechNote --verbose
[I] 17:00:03.280 0x7fb584648d00 init:49 - logging to stderr enabled
[D] 17:00:03.280 0x7fb584648d00 () - failed to load translation: "pt_BR" ":/translations"
[W] 17:00:03.280 0x7fb584648d00 () - failed to install translation
[D] 17:00:03.280 0x7fb584648d00 () - starting standalone app
[D] 17:00:03.281 0x7fb584648d00 () - app: net.mkiol dsnote
[D] 17:00:03.281 0x7fb584648d00 () - config location: "/home/carlos/.var/app/net.mkiol.SpeechNote/config"
[D] 17:00:03.281 0x7fb584648d00 () - data location: "/home/carlos/.var/app/net.mkiol.SpeechNote/data/net.mkiol/dsnote"
[D] 17:00:03.281 0x7fb584648d00 () - cache location: "/home/carlos/.var/app/net.mkiol.SpeechNote/cache/net.mkiol/dsnote"
[D] 17:00:03.281 0x7fb584648d00 () - settings file: "/home/carlos/.var/app/net.mkiol.SpeechNote/config/net.mkiol/dsnote/settings.conf"
[D] 17:00:03.281 0x7fb584648d00 () - available styles: ("Default", "Fusion", "Imagine", "Material", "org.kde.breeze", "org.kde.desktop", "Plasma", "Universal")
[D] 17:00:03.281 0x7fb584648d00 () - style paths: ("/usr/lib/qml/QtQuick/Controls.2")
[D] 17:00:03.281 0x7fb584648d00 () - switching to style: "org.kde.desktop"
[D] 17:00:03.454 0x7fb584648d00 () - supported audio input devices:
ALSA lib ../../oss/pcm_oss.c:397:(_snd_pcm_oss_open) Cannot open device /dev/dsp
[D] 17:00:03.463 0x7fb584648d00 () - "pulse"
[D] 17:00:03.503 0x7fb584648d00 () - "default"
ALSA lib ../../../src/pcm/pcm_direct.c:2045:(snd1_pcm_direct_parse_open_conf) The field ipc_gid must be a valid group (create group audio)
ALSA lib ../../../src/pcm/pcm_direct.c:2045:(snd1_pcm_direct_parse_open_conf) The field ipc_gid must be a valid group (create group audio)
[D] 17:00:03.507 0x7fb584648d00 () - "alsa_input.usb-Jieli_Technology_UACDemoV1.0-00.mono-fallback"
[D] 17:00:03.507 0x7fb584648d00 () - "alsa_output.pci-0000_09_00.4.analog-stereo.monitor"
[D] 17:00:03.508 0x7fb584648d00 add_cuda_devices:226 - scanning for cuda devices
[D] 17:00:03.508 0x7fb584648d00 add_cuda_devices:235 - cuda version: driver=0, runtime=12020
[D] 17:00:03.508 0x7fb584648d00 add_cuda_devices:240 - cudaGetDeviceCount returned: 35
[D] 17:00:03.508 0x7fb584648d00 add_hip_devices:263 - scanning for hip devices
[D] 17:00:03.508 0x7fb584648d00 hip_api:170 - failed to open hip lib: libamdhip64.so: não é possível abrir arquivo compartilhado: Arquivo ou diretório inexistente
[D] 17:00:03.508 0x7fb584648d00 add_opencl_devices:300 - scanning for opencl devices
`

**Comment by mkiol** [comment]
@caslubas Thanks for the report. What GPU is in your system?

**Comment by mutageneral** [comment]
I have the AMD ATI Radeon RX Vega M GH and Intel HD Graphics 630

```
Qt: Session management error: Could not open network socket
[I] 04:42:06.903 0x7f46e9017d80 init:49 - logging to stderr enabled
[D] 04:42:06.903 0x7f46e9017d80 () - translation: "fr_FR"
[D] 04:42:06.903 0x7f46e9017d80 () - starting standalone app
[D] 04:42:06.904 0x7f46e9017d80 () - app: net.mkiol dsnote
[D] 04:42:06.904 0x7f46e9017d80 () - config location: "/home/ordici/.var/app/net.mkiol.SpeechNote/config"
[D] 04:42:06.904 0x7f46e9017d80 () - data location: "/home/ordici/.var/app/net.mkiol.SpeechNote/data/net.mkiol/dsnote"
[D] 04:42:06.904 0x7f46e9017d80 () - cache location: "/home/ordici/.var/app/net.mkiol.SpeechNote/cache/net.mkiol/dsnote"
[D] 04:42:06.904 0x7f46e9017d80 () - settings file: "/home/ordici/.var/app/net.mkiol.SpeechNote/config/net.mkiol/dsnote/settings.conf"
[D] 04:42:06.905 0x7f46e9017d80 () - available styles: ("Default", "Fusion", "Imagine", "Material", "org.kde.breeze", "org.kde.desktop", "Plasma", "Universal")
[D] 04:42:06.905 0x7f46e9017d80 () - style paths: ("/usr/lib/qml/QtQuick/Controls.2")
[D] 04:42:06.905 0x7f46e9017d80 () - switching to style: "org.kde.desktop"
[D] 04:42:07.235 0x7f46e9017d80 () - supported audio input devices:
ALSA lib ../../oss/pcm_oss.c:397:(_snd_pcm_oss_open) Cannot open device /dev/dsp
[D] 04:42:07.250 0x7f46e9017d80 () - "pulse"
[D] 04:42:07.290 0x7f46e9017d80 () - "upmix"
[D] 04:42:07.329 0x7f46e9017d80 () - "default"
ALSA lib ../../../src/pcm/pcm_direct.c:2045:(snd1_pcm_direct_parse_open_conf) The field ipc_gid must be a valid group (create group audio)
ALSA lib ../../../src/pcm/pcm_direct.c:2045:(snd1_pcm_direct_parse_open_conf) The field ipc_gid must be a valid group (create group audio)
[D] 04:42:07.336 0x7f46e9017d80 () - "alsa_input.usb-046d_C922_Pro_Stream_Webcam_BA7C699F-02.analog-stereo"
[D] 04:42:07.336 0x7f46e9017d80 () - "alsa_output.pci-0000_01_00.1.hdmi-stereo-extra2.monitor"
[D] 04:42:07.336 0x7f46e9017d80 () - "alsa_output.usb-Apple__Inc._USB-C_to_3.5mm_Headphone_Jack_Adapter_DWH22430Y77JKLTAF-00.analog-stereo.monitor"
[D] 04:42:07.336 0x7f46e9017d80 () - "alsa_output.pci-0000_00_1f.3.analog-stereo.monitor"
[D] 04:42:07.336 0x7f46e9017d80 () - "alsa_input.pci-0000_00_1f.3.analog-stereo"
[D] 04:42:07.336 0x7f46e9017d80 add_cuda_devices:226 - scanning for cuda devices
[D] 04:42:07.337 0x7f46e9017d80 add_cuda_devices:235 - cuda version: driver=0, runtime=12020
[D] 04:42:07.337 0x7f46e9017d80 add_cuda_devices:240 - cudaGetDeviceCount returned: 35
[D] 04:42:07.338 0x7f46e9017d80 add_hip_devices:263 - scanning for hip devices
[D] 04:42:07.338 0x7f46e9017d80 hip_api:170 - failed to open hip lib: libamdhip64.so: Ne peut ouvrir le fichier d'objet partagé: Aucun fichier ou dossier de ce type
[D] 04:42:07.338 0x7f46e9017d80 add_opencl_devices:300 - scanning for opencl devices
free(): invalid size
```

**Comment by mkiol** [comment]
It looks that app crashes when doing OpenCL check.

```
[D] 04:42:07.338 0x7f46e9017d80 add_opencl_devices:300 - scanning for opencl devices
```

Can you try "beta" release from `flatpak-beta`? Follow [this manual](https://discourse.flathub.org/t/how-to-use-flathub-beta/2111) to install it. Most likely "beta" will crash as well but in "beta" I've added extra command line option `--gpu-scan-off` to disable everything related to GPU, including OpenCL.

Can you please:

- check if the problem occurs on "beta" 
- if nothing changes, run the app with `--gpu-scan-off` (only once)

**Comment by mkiol** [comment]
This problem might be fixed in the latest 4.3.0 release. If not, please re-open. Thanks.
